src/page-response-impact/ResponseImpactDashboard.tsx

```diff
@@ -113,25 +116,26 @@ const ResponseImpactDashboard: React.FC<Props> = ({
           />
           {populationFormSubmitted && (
             <ValidDataWrapper facilities={facilities.data}>
```

Comment:
I think the linter error you're seeing is because the `ValidDataWrapper` only expects 1 child element, not an array of elements. I think if you add [react fragments](https://reactjs.org/docs/fragments.html#short-syntax) around the content it should remove the error:
```
<>
  <PageHeaderContainer> ...</PageHeaderContainer>
  <PageContainer>...</PageContainer>
</>
```

Reply:
@daschi thank you, that makes a lot of sense now! I was on the wrong route thinking it was a TS thing. Much appreciated!